docs/ja/config_options.md

```diff
@@ -358,7 +358,7 @@ QMK での全ての利用可能な設定にはデフォルトがあります。
 これらを使って特定の機能のビルドを有効または無効にします。有効にすればするほどファームウェアが大きくなり、MCU には大きすぎるファームウェアを構築するリスクがあります。
 
 * `BOOTMAGIC_ENABLE`
-  * 仮想 DIP スイッチ設定
+  * Bootmagic Lite を有効にする
```

Comment:
For consistency with the rest of the translation, do the following.
```suggestion
  * ブートマジックライトを有効にします
```

Reply:
Edited in [`ec4dbb9`](https://github.com/qmk/qmk_firmware/pull/15002/commits/ec4dbb9725242bec4b70559de8f248ca07444296).